portfolio/src/main/webapp/style.css

```diff
@@ -1,28 +1,98 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.content {
+  background-color: #E1f0C4;
+  border: 2px solid #79BEEE;
+  border-radius: 5px;
+}
+
+#intro-box {
```

Comment:
where is this style used?

Reply:
It's used in index.html -- I'll create separate style sheets for each .html file.